python/gtsam/__init__.py

```diff
@@ -1,6 +1,12 @@
-from . import utils
-from .gtsam import *
-from .utils import findExampleDataFile
+"""Module definition file for GTSAM"""
+
+# pylint: disable=import-outside-toplevel, global-variable-not-assigned, possibly-unused-variable, import-error, import-self
+
+import sys
+
+from gtsam import gtsam, utils
+from gtsam.gtsam import *
```

Comment:
thanks for the PR @varunagrawal. While we're updating this, we probably want to remove this wildcard import too.

[Wildcard imports discussed in PEP8](https://www.python.org/dev/peps/pep-0008/)

Reply:
This is needed, and I think is generally OK for pybind11 modules.